### ISSUE TITLE
How many GPUs  dose the training process consume

### ISSUE THREAD

**Issue by STRUGGLE1999** [aim-uofa/Poseur#16]
Hello, I wonder to know how many gpus did you use? and the experiment equipment? The article does not mention them, thank you for your reply. And by the way, how much time does one eopch consume?

**Comment by YongtaoGe** [comment]
We use 8 gpus for training for our experiments. However, if you do not have enough resources, you can use fewer gpus (adjust learning rate and batchsize, accordingly).
